src/views/Books/BookDetail.js

```diff
@@ -1,9 +1,13 @@
 import { useEffect, useState } from 'react';
 import Book from '../../components/book/Book';
+import { useParams } from 'react-router-dom';
 import { getBookById } from '../../services/books';
+import { NavLink } from 'react-router-dom';
 
 function BookDetail() {
-  const id = 1; // TODO: Use id from route
+  const params = useParams();
+
+  const id = params.id; // TODO: Use id from route
```

Comment:
You can also destructure the id from the useParams hook to make this a one liner: const { id } = useParams();